parent/pom.xml

```diff
@@ -66,7 +66,7 @@
     <version.httpcomponents>4.4.16</version.httpcomponents>
     <version.identity>8.8.0-alpha8</version.identity>
     <version.instancio>5.5.0</version.instancio>
-    <version.surefire>3.5.3</version.surefire>
+    <version.surefire>3.5.4</version.surefire>
```

Comment:
🔧 I think we can also remove this and replace the usage of it with `plugin.version.surefire` to ensure we only specify it once.